src/Storage/AssetMapCache.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Torr\Assets\Storage;
+
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Input\ArrayInput;
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Torr\Assets\Command\AssetsDumpCommand;
+
+final class AssetMapCache
+{
+	private const CACHE_KEY = '21torr.asset.asset_map';
+
+	private CacheItemPoolInterface $cachePool;
+
+	private KernelInterface $kernel;
+
+	private AssetStorage $storage;
+
+	/**
+	 * AssetMapCache constructor.
+	 */
+	public function __construct(
+		CacheItemPoolInterface $cachePool,
+		KernelInterface $kernel,
+		AssetStorage $storage
+	)
+	{
+		$this->cachePool = $cachePool;
+		$this->kernel = $kernel;
+		$this->storage = $storage;
+	}
+
+	/**
+	 * save AssetMap in Cache
+	 */
+	public function setMapCache(AssetMap $map) : void
+	{
+		$cacheItem = $this->cachePool->getItem(self::CACHE_KEY);
+		$cacheItem->set($map);
+		$this->cachePool->save($cacheItem);
+	}
+
+	/**
+	 * get AssetMap from Cache
+	 */
+	public function getMapCache() : ?AssetMap
+	{
+		$cacheItem = $this->cachePool->getItem(self::CACHE_KEY);
+
+		if($cacheItem->isHit()) {
+			return $cacheItem->get();
+		}
+
+		return null;
+	}
+
+	/**
+	 * get AssetMap
+	 * in debug mode return new empty AssetMap and clear dump directory
+	 */
+	public function getAssetMap() : ?AssetMap
+	{
+		if($this->kernel->isDebug() === true) {
```

Comment:
```suggestion
		if ($this->kernel->isDebug()) {
```
An `if` already expects a `bool`, so no need to transform a `bool` to a `bool`.

Reply:
Changes are made in a new branch